src/main/java/com/mraof/minestuck/Minestuck.java

```diff
@@ -133,6 +137,29 @@ private void mainThreadSetup()
 		
 		ConsortDialogue.init();
 		
+		Sheets.addWoodType(MSWoodTypes.CARVED);
+		Sheets.addWoodType(MSWoodTypes.DEAD);
+		Sheets.addWoodType(MSWoodTypes.END);
+		Sheets.addWoodType(MSWoodTypes.FROST);
+		Sheets.addWoodType(MSWoodTypes.GLOWING);
+		Sheets.addWoodType(MSWoodTypes.RAINBOW);
+		Sheets.addWoodType(MSWoodTypes.SHADEWOOD);
+		Sheets.addWoodType(MSWoodTypes.TREATED);
+		Sheets.addWoodType(MSWoodTypes.LACQUERED);
+		
+		Sheets.addWoodType(MSWoodTypes.BLOOD);
+		Sheets.addWoodType(MSWoodTypes.BREATH);
+		Sheets.addWoodType(MSWoodTypes.DOOM);
+		Sheets.addWoodType(MSWoodTypes.HEART);
+		Sheets.addWoodType(MSWoodTypes.HOPE);
+		Sheets.addWoodType(MSWoodTypes.LIFE);
+		Sheets.addWoodType(MSWoodTypes.LIGHT);
+		Sheets.addWoodType(MSWoodTypes.MIND);
+		Sheets.addWoodType(MSWoodTypes.RAGE);
+		Sheets.addWoodType(MSWoodTypes.SPACE);
+		Sheets.addWoodType(MSWoodTypes.TIME);
+		Sheets.addWoodType(MSWoodTypes.VOID);
```

Comment:
Please extract these to their own method/function, then you can call that method here. Just to not clutter as much

Reply:
Yeah idk why I didn't think of just putting a method in MSWoodTypes sooner.